static/app/views/settings/organizationAuth/organizationAuthList.spec.tsx

```diff
@@ -64,14 +66,12 @@ describe('OrganizationAuthList', function () {
     });
 
     it('renders with saml available', function () {
```

Comment:
nit: why isn't SAML capitalized (or SSO below)

Reply:
Not sure